### PR TITLE
Accessibility changes

### PR DESCRIPTION
**Pull request by davidjamesstone**

### CODE REVIEW

File: test/integration/narrow/routes/incident/details-of-the-smell.test.js [hunk 1]
```diff
@@ -25,11 +25,11 @@ describe('Experience test', () => {
...
-    mockSession = {
-      description: 'test description',
-      date: '2017-08-25T00:00:00.000Z',
-      time: '14:35'
-    }
+    // mockSession = {
+    //   date: new Date().toISOString(),
+    //   hour: 14,
+    //   minute: 35
+    // }
```
Comment (paulsimonandrews) [inline test/integration/narrow/routes/incident/details-of-the-smell.test.js:32]: Remove commented out code if not needed?

**Review by paulsimonandrews — CHANGES_REQUESTED** [review]
Looks good. Small thing: validation on minute should have a min of 0 not 1, and there is some commented out code you could remove if it's not needed.

Also, just an observation, should the "County" be a required field in the address? Post doesn't require it, and many city dwellers don't use a county in their address (unless they are forced to!)

File: app/models/details-of-the-smell.js [hunk 2]
```diff
@@ -1,65 +1,137 @@
...
+  'string.empty': 'Enter the minute you noticed the smell',
+  'string.pattern.base': 'The time of day you noticed the smell must be a valid time'
+}
+
 const schema = joi.object().keys({
-  [DESCRIPTION_KEY]: joi.string().max(DESCRIPTION_LENGTH).label(DESCRIPTION_LABEL).trim().required().allow('').messages(DESCRIPTION_MESSAGES),
   [DATE_KEY]: joi.date().less('now').label(DATE_LABEL).required().messages(DATE_MESSAGES),
-  [TIME_KEY]: joi.string().regex(TIME_PATTERN).label(TIME_LABEL).required().messages(TIME_MESSAGES)
+  [TIME_HOUR_KEY]: joi.number().min(0).max(23).label(TIME_HOUR_LABEL).required().messages(TIME_HOUR_MESSAGES),
+  [TIME_MINUTE_KEY]: joi.number().min(1).max(59).label(TIME_MINUTE_LABEL).required().messages(TIME_MINUTE_MESSAGES)
```
[the remark this answers is not in the document]
Comment (davidjamesstone) [reply app/models/details-of-the-smell.js:38]: Great spot - thank you

File: test/integration/narrow/routes/incident/details-of-the-smell.test.js (hunk 1 above)
Comment (davidjamesstone) [reply test/integration/narrow/routes/incident/details-of-the-smell.test.js:32]: Yes

**Comment by davidjamesstone** [comment]
> Looks good. Small thing: validation on minute should have a min of 0 not 1, and there is some commented out code you could remove if it's not needed.
> 
> Also, just an observation, should the "County" be a required field in the address? Post doesn't require it, and many city dwellers don't use a county in their address (unless they are forced to!)

I was in two minds here. At first it wasn't mandatory, following the advice [here](https://alliescomputing.com/knowledge-base/how-to-build-a-uk-address-form). GDS guidelines aren't clear. I think you're right though - I'll raise it at standup later.

**Review by paulsimonandrews — APPROVED** [review]

**Pull request merged**